hack/update-codegen.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TEMP_DIR=`mktemp -d`
+SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
+CODEGEN_PACKAGE=$TEMP_DIR/code-generator
+
+git clone -b kubernetes-1.13.1 https://github.com/kubernetes/code-generator $CODEGEN_PACKAGE
```

Comment:
What is `kubernetes-1.13.1` here? Is it the kubernetes version the generated code targets? Currently we are (for reasons) targetting 1.9 and up. Would this still work in 1.9? If not, it should probably be documented in the README that this library targets a specific kubernetes version that is more strict than FIAAS in general.